docs/flink-writes.md

```diff
@@ -270,4 +270,4 @@ INSERT INTO tableName /*+ OPTIONS('upsert-enabled'='true') */
 ...
 ```
 
-Check out all the options here: [write-options](/flink-configuration#write-options) 
```

Comment:
nit: why these are shown as modified? Shall we revert the unrelated changed?

Reply:
Weird. Let me check and clean this.

Reply:
I used vim. It's maybe my vimrc :)  I gonna clean this ;)